src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -346,7 +346,7 @@ public boolean isClosingWithinTimeLimit(int hours) {
     }
     
     /**
-     * returns true if the session is closed within the past hour of calling this function
+     * @return true if the session is closed within the past hour of calling this function
```

Comment:
Sadly you have been proven wrong, since `SingleLineJavadoc` is already in action.